notes/ltc-transfer-penalty.vbs

```diff
@@ -67,25 +69,27 @@ BeginDialog Dialog1, 0, 0, 146, 70, "Case number dialog"
   ButtonGroup ButtonPressed
     OkButton 35, 45, 50, 15
     CancelButton 90, 45, 50, 15
-  Text 10, 30, 65, 10, "Footer month/year:"
-  Text 10, 10, 45, 10, "Case number: "
+  Text 35, 10, 45, 10, "Case number: "
+  Text 15, 30, 65, 10, "Footer Month/Year:"
 EndDialog
 
 DO
-	DO
-		err_msg = ""							'establishing value of variable, this is necessary for the Do...LOOP
-		dialog Dialog1				'main dialog
-		cancel_without_confirmation
-		IF len(MAXIS_case_number) > 8 or isnumeric(MAXIS_case_number) = false THEN err_msg = err_msg & vbCr & "* Enter a valid case number."		'mandatory fields
-        If IsNumeric(MAXIS_footer_month) = False or len(MAXIS_footer_month) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a valid 2-digit MAXIS footer month."
-        If IsNumeric(MAXIS_footer_year) = False or len(MAXIS_footer_year) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a valid 2-digit MAXIS footer year."
-		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
-	LOOP UNTIL err_msg = ""									'loops until all errors are resolved
+    DO
+        err_msg = ""							'establishing value of variable, this is necessary for the Do...LOOP
+        dialog Dialog1				'main dialog
+        cancel_without_confirmation
+        Call validate_MAXIS_case_number(err_msg, "*")
+        Call validate_footer_month_entry(MAXIS_footer_month, MAXIS_footer_year, err_msg, "*")
+        IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
+    LOOP UNTIL err_msg = ""									'loops until all errors are resolved
 	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 Loop until are_we_passworded_out = false					'loops until user passwords back in
 
 'information gathering to auto-populate LTC_transfer_penalty_dialog
 'grabbing app date from STAT/PROG
+
+Call check_for_MAXIS(False) 'checking for an active MAXIS session
+back_to_SELF
 Call navigate_to_MAXIS_screen("STAT", "PROG")
```

Comment:
I tested this on a case that did not have HC active/pending, so we might think about putting in a data validation step in here to ensure that HC is either active or pending.